addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -371,7 +371,17 @@ export const editorCommands = {
     italic: editor => editor.document.execCommand('italic'),
     underline: editor => editor.document.execCommand('underline'),
     strikeThrough: editor => editor.document.execCommand('strikeThrough'),
-    removeFormat: editor => editor.document.execCommand('removeFormat'),
+    removeFormat: editor => {
+        editor.document.execCommand('removeFormat');
+        const selectedNodes = [...new Set(getTraversedNodes(editor.editable))];
+        for (let node of selectedNodes) {
+            if (node.nodeType === Node.TEXT_NODE) {
+                node = node.parentNode;
+            }
```

Comment:
This is done so often we actually have an util for it so you can do:
```js
closestElement(node).style.backgroundImage = '';
```
and node can be a `const`.